day_7/fizzbuzz.js

```diff
@@ -0,0 +1,14 @@
+for (var i=1; i<= 100; i++){
```

Comment:
Nice job on this Fizzbuzz challenge! Again, it's important to pay attention to detail on the instructions, as the output of `Fizz`, `Buzz`, and `FizzBuzz` were meant _to be in place of_ the number output itself, not in addition to.

Reply:
oops, fixed!